### ISSUE TITLE
Setting the quantity on the SubscriptionBuilder only works when the user has a mandate

### ISSUE THREAD

**Issue by sebastiaands** [laravel/cashier-mollie#70]
When I create a new subscription it looks like I can set the quantity on the `SubscriptionBuilder`:
`$user->newSubscription($name, $plan)->quantity(10)->create();`

This does work fine when the user already has a mandate, but is doesn't work when the payment is done using iDEAL.

**Comment by sandervanhooft** [comment]
Hi @sebastiaands ,

That's unfortunate. Can you share the payment 1) metadata and 2) amount? It's in the Mollie Dashboard.

**Comment by sebastiaands** [comment]
Hi @sandervanhooft ,

Thank you for your response.
Hereby an example of the metadata:
```
{
  "owner": {
    "type": "App\\Organisation",
    "id": 34
  },
  "actions": [
    {
      "handler": "Laravel\\Cashier\\FirstPayment\\Actions\\StartSubscription",
      "description": "Abonnement voor één expertise",
      "subtotal": {
        "currency": "EUR",
        "value": "175.00"
      },
      "taxPercentage": "21.0000",
      "plan": "one_expertise",
      "name": "partners",
      "quantity": 4
    }
  ]
}
```

**Comment by sandervanhooft** [comment]
Thanks!

Preparing a fix in #71

**Comment by sandervanhooft** [comment]
Just released [a fix](https://github.com/laravel/cashier-mollie/releases/tag/v1.2.0)
